packages/create-content-sdk-app/src/templates/nextjs/sitecore.cli.config.ts

```diff
@@ -1,17 +1,22 @@
-import config from './sitecore.config';
+import scConfig from './sitecore.config';
 import { defineCliConfig } from '@sitecore-content-sdk/nextjs/config-cli';
-import { generateSites, generateMetadata, extractFiles } from '@sitecore-content-sdk/nextjs/tools';
+import { generateSites, generateMetadata, extractFiles, writeImportMap } from '@sitecore-content-sdk/nextjs/tools';
 
 export default defineCliConfig({
   build: {
     commands: [
       generateMetadata(),
       generateSites({
-        scConfig: config,
+        scConfig,
       }),
       extractFiles({
-        scConfig: config,
+        scConfig,
       }),
+      writeImportMap({
+        paths: ['src/components'],
+      },
+      scConfig
```

Comment:
If we keep scConfig - lets have passed as object property, so we can make it optional in future and provide it in centralized manner.